### PR TITLE
Frontend changes for varToTopics changes

### PR DESCRIPTION
**Pull request by juliawu**
This is a companion PR to #3445 . It adds the frontend changes that pair with the new varToTopics in fulfillment return.

Screenshot 2023-09-08 at 11 09 06 AM

### CODE REVIEW

File: experimental/sdg-tracker/src/components/countries/CountriesContent.tsx [hunk 1]
```diff
@@ -332,49 +332,50 @@ interface Goals {
...
-        const goal =
-          goalMatches && goalMatches.length > 1 ? goalMatches[1] : "none";
-
-        // put tile in appropriate spot in allGoals
-        if (goal in allGoals) {
-          if (target in allGoals[goal]) {
-            if (indicator in allGoals[goal][target]) {
-              allGoals[goal][target][indicator].push(tile);
+        if (!_.isEmpty(tile.statVarKey) && !_.isEmpty(varToTopics[tile.statVarKey[0]])) {
+          for (const topic of varToTopics[tile.statVarKey[0]]) {
```
Comment (pradh) [inline experimental/sdg-tracker/src/components/countries/CountriesContent.tsx:345]: For my understanding, will a page show the same chart now under different topics?  Or will something downstream only show a specific indicator?

**Review by pradh — APPROVED** [review]

File: experimental/sdg-tracker/src/components/countries/CountriesContent.tsx (hunk 1 above)
Comment (pradh) [reply experimental/sdg-tracker/src/components/countries/CountriesContent.tsx:345]: Ok.

If the LHS is showing Goal 1 though, it doesn't make a lot of sense to show anything related to 2.2 on RHS right?

Comment (pradh) [reply experimental/sdg-tracker/src/components/countries/CountriesContent.tsx:345]: (Not for today)

Comment (juliawu) [reply experimental/sdg-tracker/src/components/countries/CountriesContent.tsx:345]: Agreed, let's get this change in and I'll add more filtering in a followup PR.

**Pull request merged**